prefect_docker/scripts/configuration/settings.toml

```diff
@@ -1,30 +1,38 @@
 [default]
+ELSST_SKOSMOS_URL="https://thesauri.cessda.eu"
 
 [development]
 debug = true
-
-DATAVERSE_URL="http://dataverse:8080"
-
+DOI_MINTER_URL = "http://datacite-minter:9999"
```

Comment:
The readme keeps track of each service and its port. Please add this one to the readme.

Reply:
Fixed.